README.md

```diff
@@ -0,0 +1,20 @@
+# Welcome to Anypoint Code Builder Beta!
+
+Anypoint Code Builder is a web-based integrated development environment (IDE) that has all the power and flexibility of Visual Studio Code along with the Anypoint Code Builder extensions. Anypoint Code Builder provides a modern developer experience for all developers to design, develop, and deploy APIs, integrations, and automations from a single environment.
+
+This public repository is your place to give us feedback or report an issue.
+
+## We Want Your Feedback
+
+We really appreciate your participation in the Anypoint Code Builder beta. Your feedback will help shape the future of Anypoint Code Builder.
+
+To send us feedback simply create [an issue](https://github.com/forcedotcom/try-acb-feedback/issues) in this repository. We are looking for general impressions and feedback on how this tool helps you in your current workflow, feature requests, and bugs. There are templates for each type of feedback. We look forward to hearing from you!
```

Comment:
>There are templates for each type of feedback

I took this from Salesforce Code Builder's repository. Is this true for us as well @simoneegeib ?

Reply:
Yes, it will be. I'll work with the salesforce PM to create those templates